src/translations/en.json

```diff
@@ -107,6 +107,10 @@
         "off": "[%key:state::device_tracker::not_home%]",
         "on": "[%key:state::device_tracker::home%]"
       },
+      "problem": {
+        "off": "[%key:state::plant::ok%]",
```

Comment:
Shouldn't this be under `binary_sensor` ?

Reply:
Yeah, plant became the master since this didn't exist at the time, but this would have been the master otherwise. I'll reorganize it.